pkg/endpointmanager/cell.go

```diff
@@ -106,7 +107,7 @@ type EndpointManager interface {
 
 	// InitMetrics hooks the EndpointManager into the metrics subsystem. This can
 	// only be done once, globally, otherwise the metrics library will panic.
-	InitMetrics()
+	InitMetrics(regitry *metrics.Registry)
```

Comment:
Why we need to add `InitMetrics` to the `EndpointManager` interface?

Nit: typo on the param name `regitry` -> `registry`

Reply:
That is a very good point. I did not think of that. It was likely added to the signature so it matches exactly, but it is never actually called on the interface type as far as I can see. So removed it :rocket: